### PR TITLE
Updated about page text and images

### LINKED ISSUE #561: About: new images and a few more typos
#### Designs and Mockups

See the updated text at https://docs.google.com/document/d/1HUivXOfS1bovVAJX2u_2QERqUSlU9dG2uQiC0vVIqDA/edit#

new images are in https://drive.google.com/drive/u/0/folders/1mAJf6Jcr1nZrCtwb3g9_KOuQ14mMelXI or you can get them from the ocw-studio links below: 

#### Acceptance Criteria:

- [ ] Remove extra space after `...`
https://github.com/mitodl/ocw-hugo-themes/blob/59e9a37c4840e1814088ea962ea73562b5be35a1/www/layouts/about/section.html#L184
```suggestion
            and insightful ways to discover the content you need…and will
```
- [ ] New image for 2001-2005 milestones: [2001_ChuckVest.png](https://ocw-studio.odl.mit.edu/sites/ocw-www/type/resource/edit/4dda1ab4-119c-4c08-a820-81c7859a5c56/)
- [ ] remove "OCW Highlights for High School launched."
- [ ] New image for 2006-2010 milestones: [2007_1800_OCWsite](https://ocw-studio.odl.mit.edu/sites/ocw-www/type/resource/edit/00ef8b32-e055-47aa-8e32-4a9f04be0920/)
- [ ] "225 OCW mirror sites are installed around the world."
https://github.com/mitodl/ocw-hugo-themes/blob/59e9a37c4840e1814088ea962ea73562b5be35a1/www/content/milestones/2006-2010.md#L10
```suggestion
            - 225 OCW mirror sites are installed around the world.
```
- [ ] remove "Publish OCW Scholar courses, designed for independent learners."
- [ ] new image for 2011-2015 milestones: [2015_Educator.png](https://ocw-studio.odl.mit.edu/sites/ocw-www/type/resource/edit/8f8d2eb6-b465-495b-9fd6-7b0c61940245/)
- [ ] new image for 2016-2020 milestones: [2018_YouTube.png](https://ocw-studio.odl.mit.edu/sites/ocw-www/type/resource/edit/07bc04ae-1b82-4135-ae54-10d60306fb90/)
- [ ] in 2021+, add "Over 300 million lifetime visits to OCW website and YouTube channel"
- [ ] new image for 2021+ [2021_NextGen_phone.png](https://ocw-studio.odl.mit.edu/sites/ocw-www/type/resource/edit/455f6fa8-0216-448c-986f-b2edf16cf86d/)
- [ ] link to "Find courses with Instructor Insights" should be https://ocwnext.odl.mit.edu/search/?f=Instructor%20Insights&s=-runs.best_start_date

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
- [x] Testing
    - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #561 

#### What's this PR do?
Updates About page text and images as per [doc](https://docs.google.com/document/d/1HUivXOfS1bovVAJX2u_2QERqUSlU9dG2uQiC0vVIqDA/edit#) 

#### How should this be manually tested?
Verify all changes have been accommodated as per issue #561  

#### Screenshots (if appropriate)
Screenshot 2022-03-29 at 4 14 51 PM

### CODE REVIEW

**Comment by Wassaf-Shahzad** [comment]
@pdpinch  i couldn't find the required image for 2021+. Would you kindly share the link. Everything else except the 2021+ images has been completed.

**Comment by pdpinch** [comment]
The 2021+ image is https://drive.google.com/file/d/18qS5V2PIf2IxYNr_zpll2B3T7n1NRjMM/view?usp=sharing

**Comment by pdpinch** [comment]
Can you also change the link to "/pages/open-learning-library/" to go to "/course-lists/open-learning-library/" instead ?

**Comment by Wassaf-Shahzad** [comment]
> Can you also change the link to "/pages/open-learning-library/" to go to "/course-lists/open-learning-library/" instead ?

Done

**Comment by pdpinch** [comment]
This might be out of scope for this PR, but I just noticed that the staff photos disappear entirely in the mobile layout (I tested firefox and safari).

Screen Shot 2022-03-31 at 08 06 11

Let me know if you can quickly fix this here, or if it should be a separate issue.

**Comment by pdpinch** [comment]
Two more issues in mobile: 

- [ ] The left margin for the red box with the text "In our first 20 years" is too small: 

image

- [ ] The text on the button "Find Courses with Instructor Insights" is getting wrapped when it does need to be:

image

**Review by pdpinch — CHANGES_REQUESTED** [review]
some issues have cropped up in mobile

File: www/layouts/about/section.html [hunk 1]
```diff
@@ -358,7 +358,7 @@ <h1 class="text-center-mobile pb-5">Beyond Course Materials</h1>
               Open Learning Library is always open for self-guided learning, and 
               does not include live support, discussion forums, or certificates of completion.
             </p>
-            <a class="red-btn" href="/pages/open-learning-library/">Explore Open Learning Library courses </a>
+            <a class="red-btn" href="/course-lists/open-learning-library/">Explore Open Learning Library courses </a>
```
Comment (MAbdurrehman12) [inline www/layouts/about/section.html:361]: I think you can undo this change and rebase your branch with main as this link: `Explore Open Learning Library courses` was updated in this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/588) yesterday

File: www/layouts/about/section.html [hunk 2]
```diff
@@ -327,7 +327,7 @@ <h1 class="text-center-mobile pb-5">Beyond Course Materials</h1>
               practicing teachers, covering topics such as course design, active 
               learning methods, and strategies to engage learners.
             </p>
-            <a class="red-btn" href="/search/?type=Instructor%20Insights&s=-runs.best_start_date"
+            <a class="red-btn" href="https://ocwnext.odl.mit.edu/search/?f=Instructor%20Insights&s=-runs.best_start_date"
```
Comment (MAbdurrehman12) [inline www/layouts/about/section.html:330]: I think we should do it something like `href="/search/...` instead of `href="https://ocwnext.odl.mit.edu/...` as currently prod is on `https://ocw.mit.edu...`

File: www/layouts/about/section.html (hunk 1 above)
Comment (MAbdurrehman12) [reply www/layouts/about/section.html:361]: I think you can undo this change as the correct link was provided by Daniel (as shown below) for which a PR (mentioned in above comment) was created and merged:
image

File: www/content/milestones/2021.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
 title: "2021+"
-image: /images/about/timeline-image-1.png
+image: /images/about/2021_phone.png
 headless: true
 milestones:
   - OCW celebrates its 20th anniversary and two decades of open sharing ([video](https://www.youtube.com/watch?v=8O2EPwUtWyY)).
   - "New OCW platform launches: mobile responsive, enhanced search and more"
-  - 440 OCW mirror sites around the world.
+  - 440 OCW mirror sites are installed around the world.
+  - Over 300 million lifetime visits to OCW website and YouTube channel
```
Comment (MAbdurrehman12) [inline www/content/milestones/2021.md:9]:
```suggestion
  - Over 300 million lifetime visits to OCW website and YouTube channel.
```

**Review by MAbdurrehman12 — CHANGES_REQUESTED** [review]
Looks good.
Few things which I noticed: 

1. I think there have been some changes in this [document](https://docs.google.com/document/d/1BKM1yBGjXiHBMVU4DeUiG4RuPZ0jbZ-j/edit) and the current content does not seems to be up-to-date with that.
2. The changes mentioned in this [document](https://docs.google.com/document/d/1HUivXOfS1bovVAJX2u_2QERqUSlU9dG2uQiC0vVIqDA/edit#) are also pending I think (maybe the doc has been updated a couple of days ago)

**Comment by MAbdurrehman12** [comment]
> Can you also change the link to "/pages/open-learning-library/" to go to "/course-lists/open-learning-library/" instead ?

@pdpinch This was updated to `/collections/...` in this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/588) on launch day. ([Slack thread](https://mitodl.slack.com/archives/GQ89D0Z5M/p1648737315939519) for reference).
I think we can undo this change from here

**Comment by MAbdurrehman12** [comment]
@Wassaf-Shahzad Sorry, I forgot to mention one thing. 
This sub-navbar:
image
is not working for on my local setup, neither for about page nor for educator page. 
The buttons are not being clicked nor being highlighted on scroll, though I think there is something wrong at my end but you can also confirm this

File: www/layouts/about/section.html (hunk 2 above)
Comment (pdpinch) [reply www/layouts/about/section.html:330]: Yes!

**Comment by pdpinch** [comment]
> I think there have been some changes in this [document](https://docs.google.com/document/d/1BKM1yBGjXiHBMVU4DeUiG4RuPZ0jbZ-j/edit) and the current content does not seems to be up-to-date with that.
>    The changes mentioned in this [document](https://docs.google.com/document/d/1HUivXOfS1bovVAJX2u_2QERqUSlU9dG2uQiC0vVIqDA/edit#) are also pending I think (maybe the doc has been updated a couple of days ago)

This is correct. There have been a number of text changes on 4/1. I'll try to organize them to make them easier to work with.

File: www/layouts/about/section.html (hunk 2 above)
Comment (Wassaf-Shahzad) [reply www/layouts/about/section.html:330]: Yes I did that in my first update but the second round of updated suggested the full link as you can see in the diff so I updated it accordingly.
I will revert it back

**Comment by Wassaf-Shahzad** [comment]
> > Can you also change the link to "/pages/open-learning-library/" to go to "/course-lists/open-learning-library/" instead ?
> 
> @pdpinch This was updated to `/collections/...` in this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/588) on launch day. ([Slack thread](https://mitodl.slack.com/archives/GQ89D0Z5M/p1648737315939519) for reference). I think we can undo this change from here

I can try to rebase again and pull latest changes from master but I dont think we should undo this change

File: www/layouts/about/section.html (hunk 1 above)
Comment (pdpinch) [reply www/layouts/about/section.html:361]: Please use https://ocw.mit.edu/collections/mit-open-learning-library/

**Comment by Wassaf-Shahzad** [comment]
@pdpinch  @MAbdurrehman12  Updated text

**Comment by pdpinch** [comment]
Thanks for all your work on this. I got a 👍 from the author, so let's merge it. 

There may be more changes in the future, but it's good enough for now.

Maybe we'll get a chance to turn this into a proper template, so we can get out of the business of text changes.

**Review by MAbdurrehman12 — APPROVED** [review]
Looks good 👍
